sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/implementation/models/KeyRotationPolicy.java

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.implementation.models;
+
+import com.azure.core.annotation.Fluent;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * Represents the rotation policy for a key.
+ */
+@Fluent
+public final class KeyRotationPolicy {
+    @JsonProperty(value = "id", access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
What do `access` do? I noticed it's only on id and am curious

Reply:
According to Jackson (our serialization/deserialization library):

> . . . property that may be used to change the way visibility of accessors (getter, field-as-getter) and mutators (constructor parameter, setter, field-as-setter) is determined, either so that otherwise non-visible accessors (like private getters) may be used; or that otherwise visible accessors are ignored . . . specifies how a property may be accessed during serialization ("read") and deserialization ("write").

For example, the `WRITE_ONLY` property:

> . . . means that the property may only be written (set) as part of deserialization (using "setter" method, or assigning to Field, or passed as Creator argument) but will not be read (get) for serialization, that is, the value of the property is not included in serialization.

Reply:
Cool, thank you!